experimental/dds/tree/README.md

```diff
@@ -0,0 +1,206 @@
+
+Say we wanted to delete the `fooNode` we inserted above. There are 4 ways we could specify the `StableRange` to delete which are all equivalent in the absence of collaboration:
+
+```typescript
+const trait = { parent: initialTree, label: 'foo' };
+const stableRange1 = StableRange.from(StablePlace.atStartOf(trait)).to(StablePlace.atEndOf(trait));
+const stableRange2 = StableRange.from(StablePlace.atStartOf(trait)).to(StablePlace.after(fooNode));
+const stableRange3 = StableRange.from(StablePlace.before(fooNode)).to(StablePlace.atEndOf(trait));
+const stableRange4 = StableRange.from(StablePlace.before(fooNode)).to(StablePlace.after(fooNode));
+```
+
+Once collaboration is considered, the different ways to anchor this `StableRange` may impact whether or not this edit conflicts with others.
+
+Also note that there are some more convenient shorthands for several of these specifications. See `StableRange` documentation for more information.
+
+# Status
+
+SharedTree is in active, but still relatively early development. As such, it is lacking in some areas (such as performance testing). For an idea of some future features we'd like to support, see [Future Plans](./docs/Future.md).
+
+Implementation-wise:
+
+-   Document format may change only in major releases, and SharedTree is commited to backwards compatibility (support for older documents). For more information on this commitment, see the notes in [PersistedTypes.ts](./src/PersistedTypes.ts).
+-   APIs are not yet stable, and those beyond what's needed for the MVP (ex: history editing and inspection) are not provided yet. Core APIs are not likely to significantly change.
+-   Performance is generally reasonable. However, this assessment was made using integration-style performance tests of consuming applications. Though it's on the roadmap, there are currently no rigorous performance tests which are isolated to SharedTree.
+
+Design wise:
+
+-   SharedTree is always created with an uninitialized state. It is up to the application to use the shared tree to initialize the tree to something else if needed.
+-   There are still open questions regarding how SharedTree will relate to the rest of the Fluid ecosystem.
+    For example, we do not have suggested design patterns for when users of SharedTree should used references to other Fluid components versus including the data for children as subtrees.
+
+# Edits
+
+An `Edit` is the basic unit of transactionality in `SharedTree`. It specifies how to modify a document via a sequence of changes (see [PersistedTypes.ts](.\src\PersistedTypes.ts)). Each edit applied to a version of the document (a Snapshot) produces a new version of the document.
+
+Once an edit is acknowledged by Fluid (and thus it has a sequence number, and will be included in summaries), the version of the document it applies to is fixed: it will not be applied to any revision other than the one produced by its preceding edit, transitively ack to the initial tree.
```

Comment:
```suggestion
Once an edit is acknowledged by the Fluid service (and thus it has a sequence number, and will be included in summaries), the version of the document it applies to is fixed: it will not be applied to any revision other than the one produced by its preceding edit.
```

(resolving other comment as well with this suggestion)